js/Makefile

```diff
@@ -21,7 +21,7 @@ build:
 
 .PHONY: test
 test: build
-	(cd stdlib && $(MAKE) test)
+	(cd stdlib && $(MAKE) check clean-test && sbin/test.sh)
```

Comment:
Why the change on this line? These details of how stdlib test works already exist in stdlib/Makefilfe

Reply:
Ok I read the description now :P nevermind

Reply:
I addressed this in the description:

"Prevent subtle timing issue in js/Makefile vs. js/stdlib/Makefile
when invoking test target: by pulling test completely into the
parent Makefile we can ensure that a fresh runner Docker image has
already been created first (and therefore local changes make it into
test suite container)"